cypress.json

```diff
@@ -0,0 +1,13 @@
+{
+    "experimentalComponentTesting": true,
+    "componentFolder": "src/ui/components/Base",
+    "baseUrl": "http://localhost:3000",
+    "env": {
+        "auth_audience": "https://yeehaw69.us.auth0.com/api/v2/",
+        "auth_url": "https://yeehaw69.us.auth0.com/oauth/token",
+        "auth_client_id": "3q6XrgYEBQOc4dpP84R4aSMg2UTm8uV5",
+        "auth_client_secret": "mQp1QuQ8i6HAR0__q0tZ3b-6ZVU-vRDotEmiB2frLHaz-2ZAkkLJ_CZgvKTa0VJ0",
+        "auth_username": "e2e-testing@mydomain.com",
+        "auth_password": "s#m3R4nd0m-pass"
```

Comment:
Im sure there is a more secure way to store username/password, especially in a public repo. please research

Reply:
we can store environment variables in github and then get those when the application starts.